src/requests/Request.tsx

```diff
@@ -13,20 +14,53 @@ export interface RequestProps {
 
 class Request extends PureComponent<RequestProps> {
 
-  get requestEntries(): JSX.Element[] {
+  codeMapper(code: string): string {
+    if (code.length === 4) {
+      switch (code) {
+        case '1001':
+          return 'INVALID_CONTENT_TYPE';
+        case '4000':
+          return 'INVALID_SELECTOR_DATA';
+        case '4004':
+          return 'NO_MATCHING_ELEMENTS_FOUND';
+        case '5000':
+          return 'INTERNAL_SERVER_ERROR';
+        default:
+          return 'OK';
+      }
+    } else if (code.length === 3) {
+      return `HTTP ERROR ${code}`;
+    }
+    return 'OK';
+  };
+
+  render() {
     const {
       id, url, validFrom, value, errorCode,
     } = this.props;
+    console.log('====================================');
```

Comment:
please remove console log - it will be throw lines every on every render call

Reply:
done